spring-rabbit/src/test/resources/ssl.properties

```diff
@@ -2,3 +2,5 @@ keyStore=file:/path/to/client/keycert.p12
 trustStore=file:/path/to/client/truststore
 keyStore.passPhrase=secret
 trustStore.passPhrase=secret
+keyStore.type=foo
+trustStore.type=bar
```

Comment:
This is one more annoying formatting issue.
You can configure your IDE to fix that on save.

But right now it is fine and I'm merging your PR after all that formatting hacks from your side :smile: 

Anyway you still owe a description what was wrong with your formatting settings :wink: